lms/djangoapps/verify_student/views.py

```diff
@@ -1213,3 +1214,36 @@ def get(self, request):
         """
         IDV_workflow = IDVerificationService.get_verify_location()
         return redirect(IDV_workflow)
+
+
+class PhotoUrlsView(APIView):
+    """
+    This can be used to help debug IDV photos
+    """
+
+    @method_decorator(require_global_staff)
+    def get(self, request):
+        """
+        Endpoint for retrieving photo urls for IDV
+
+        Returns:
+            200 OK
+            {
```

Comment:
Can we verify this json blob in your unit test? You only checked the existence of `EdX-ID`. I would want to make sure the URLs are also sent back correctly.